lib/transactions/payment.js

```diff
@@ -25,32 +25,28 @@ function Payment(opts) {
   Transaction.call(this, opts.transaction);
 
   this.paymentAddress = new Address(opts.paymentAddress);
-  this._change = new Address(opts.changeAddress);
+  this.changeAddress = new Address(opts.changeAddress);
+  if (!this.outputs.length) {
+    this.change(this.changeAddress);
+    this._updateChangeOutput();
```

Comment:
Is this needed? `.change` calls `_updateChangeOutput()` https://github.com/bitpay/bitcore/blob/master/lib/transaction/transaction.js#L660